fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -342,90 +336,26 @@ public void replaceIndex(String newLuceneIndexPathString) {
             String tempDir = oldLuceneIndexPath.resolveSibling(oldLuceneIndexPath.getFileName() + "_old").toString();
 
             LOG.trace("Moving old Lucene index directory from {} to {} ...", luceneIndexPath, tempDir);
-            moveDirEntries(luceneIndexPath, tempDir);
+            Utils.moveDirEntries(luceneIndexPath, tempDir);
 
             LOG.trace("Moving all new Lucene indexes from {} to {} ...", newLuceneIndexPathString, luceneIndexPath);
-            moveDirEntries(newLuceneIndexPathString, luceneIndexPath);
+            Utils.moveDirEntries(newLuceneIndexPathString, luceneIndexPath);
 
             LOG.trace(
                     "Deleting {} since new Lucene indexes have been moved away from there and is now empty",
                     newLuceneIndexPathString
             );
-            deleteDir(newLuceneIndexPathString);
+            Utils.deleteFiles(newLuceneIndexPathString);
```

Comment:
`deleteDir()` is doing  the same thing as `Utils.deleteFiles`, which has already been implemented a while ago.